src/azsdkdemoprocessor/src/main/java/demo/sdk/az/App.java

```diff
@@ -0,0 +1,93 @@
+package demo.sdk.az;
+
+import ch.qos.logback.classic.Logger;
+import com.azure.identity.*;
+import com.azure.storage.queue.*;
+import com.microsoft.applicationinsights.logback.ApplicationInsightsAppender;
+import io.github.cdimascio.dotenv.*;
+import java.util.*;
+import org.slf4j.LoggerFactory;
+
+public class App {
+  static final Logger logger = (Logger) LoggerFactory.getLogger("root");
+  static final Dotenv env = Dotenv
+    .configure()
+    .ignoreIfMissing()
+    .systemProperties()
+    .directory("../../")
+    .load();
+
+  public static void main(String[] args) {
+    try {
+      App app = new App();
+      app.setInstrumentationKey();
+      app.process();
+    } catch (Exception e) {
+      logger.error("Program Execution Error", e);
+    }
+  }
+
+  private void setInstrumentationKey() {
```

Comment:
nit: can be made fluent.

Reply:
Do you have an example of what that would like and best way to implement?  (This is first java thing I've done like this)